core/deployment/src/main/java/io/quarkus/deployment/builditem/FeatureBuildItem.java

```diff
@@ -35,6 +35,7 @@
     public static final String KOTLIN = "kotlin";
     public static final String KUBERNETES = "kubernetes";
     public static final String KUBERNETES_CLIENT = "kubernetes-client";
+    public static final String LAMBDA = "lambda";
```

Comment:
Maybe having `amazon-lambda` would be better? `lambda` is too generic IMHO.

Reply:
That's done.

13:23:38 INFO  [io.quarkus]] (main) Installed features: [**amazon-lambda**, cdi, kotlin]